modules/router-store/src/actions.ts

```diff
@@ -34,6 +35,10 @@ export type RouterRequestAction<
   payload: RouterRequestPayload<T>;
 };
 
+export const routerRequestAction = createAction(
+  ROUTER_REQUEST,
+  props<{ payload: RouterRequestPayload<BaseRouterStoreState> }>()
```

Comment:
These payloads should be from the generic `SerializedRouterSnapshot` (my bad, I was wrong in the example).

```suggestion
  props<{ payload: RouterRequestPayload<SerializedRouterSnapshot> }>()
```

Reply:
I think you mean `SerializedRouterStateSnapshot` instead of `SerializedRouterSnapshot`

Reply:
Correct, thanks!